.editorconfig

```diff
@@ -40,45 +40,1453 @@ dotnet_style_require_accessibility_modifiers = omit_if_default:suggestion
 
 ### Code Style Analyzers
 
+## BCL analyzer rules
```

Comment:
Why this way? All rules are already enabled and if there are any problematic we should disable them instead of this impossible to maintain config

Reply:
This is a copy of what runtime is doing in  https://github.com/dotnet/runtime/blob/main/eng/CodeAnalysis.src.globalconfig
Eventually, after moving the linker to runtime this editor config would disappear and we would basically use the same globalconfig + a small editor config to keep warnings for things like unused imports

Reply:
I have mixed feeling about doing it this way as it will get out of sync. I think it'd be better to update/fix the sources as part of the dotnet/runtime migration.